ballerina/build.gradle

```diff
@@ -62,7 +62,7 @@ ballerina {
     packageOrganization = packageOrg
     module = packageName
     langVersion = ballerinaLangVersion
-    testCoverageParam = '--code-coverage --coverage-format=xml --includes=org.ballerinalang.jdbc.*:ballerinax.*'
+    testCoverageParam = '--code-coverage --coverage-format=xml --includes=io.ballerina.stdlib.java.jdbc.*:ballerinax.*'
```

Comment:
Can't we take the `ballerinax` part from the package org in the Gradle plugin? If so we can remove this, right?

Reply:
Yes, we can, I'll add an issue for improvement. 
However, we cant remove the filter from here since the package name consist of period.